packages/admin/src/table/filterbar/FilterBarPopOverFormField.tsx

```diff
@@ -0,0 +1,187 @@
+    | "styledBox"
+    | "labelWrapper"
+    | "popoverContentContainer"
+    | "buttonsContainer"
+    | "submitContainer"
+    | "resetCloseContainer";
+
+const useStyles = makeStyles(
+    (theme: Theme) => ({
+        root: {
+            position: "relative",
+        },
+        styledBox: {
+            position: "relative",
+            alignItems: "center",
+            padding: "10px 20px",
+            cursor: "pointer",
+            display: "flex",
+
+            "&:after": {
+                borderTop: `4px solid ${theme.palette.grey[300]}`,
+                borderRight: "4px solid transparent",
+                borderLeft: "4px solid transparent",
+                position: "absolute",
+                display: "block",
+                right: "10px",
+                content: "''",
+                top: "50%",
+                height: 0,
+                width: 0,
+            },
+        },
+        labelWrapper: {
+            marginRight: "15px",
+            boxSizing: "border-box",
+        },
+        popoverContentContainer: {
+            "& [class*='CometAdminFormFieldContainer-root']": {
+                boxSizing: "border-box",
+                padding: "20px",
+                marginBottom: 0,
+            },
+        },
+        buttonsContainer: {
+            borderTop: `1px solid ${theme.palette.grey[300]}`,
+            justifyContent: "space-between",
+            padding: "10px 15px",
+            display: "flex",
+        },
+        submitContainer: {},
+        resetContainer: {
+            marginRight: "15px",
+        },
+    }),
+    { name: "CometAdminFilterBarPopOverFormField" },
+);
+
+interface IFormFieldProps extends FieldRenderProps<any> {
+    field: IFilterBarField;
+    handleSubmit: () => void;
+}
+
+export const FilterBarPopOverFormField: React.FunctionComponent<IFormFieldProps & PopoverFormFieldThemeProps> = ({ field, dirtyFieldsBadge }) => {
+    const DirtyFieldsBadgeComponent = field.dirtyFieldsBadge ? field.dirtyFieldsBadge : FilterBarDirtyFieldsBadge;
+    const outerForm = useForm();
+    const [anchorEl, setAnchorEl] = React.useState<HTMLDivElement | null>(null);
+    const open = Boolean(anchorEl);
+
+    const handleClick = (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+        setAnchorEl(event.currentTarget);
+    };
+
+    const { values } = outerForm.getState();
+    const { props: themeProps } = useTheme<ThemeOptions>();
+    const submitButtonProps =
+        themeProps && themeProps["CometAdminFilterBarPopOverFormField"] ? { ...themeProps["CometAdminFilterBarPopOverFormField"]?.submitButton } : {};
+    const resetButtonProps =
+        themeProps && themeProps["CometAdminFilterBarPopOverFormField"] ? { ...themeProps["CometAdminFilterBarPopOverFormField"]?.resetButton } : {};
+
+    const classes = useStyles({ open: open });
+
+    return (
+        <Form
+            onSubmit={(values) => {
+                const formFieldValues = get(values, field.name);
```

Comment:
```suggestion
                const formFieldValue = get(values, field.name);
```

as this a single value